pkg/codegen/go/gen.go

```diff
@@ -809,9 +832,15 @@ func (pkg *pkgContext) genHeader(w io.Writer, goImports []string, importedPackag
 	var imports []string
 	if len(importedPackages) > 0 {
 		for k := range importedPackages {
-			imports = append(imports, k)
+			imports = append(imports, fmt.Sprintf("%s", k))
```

Comment:
Why is this change necessary?

Reply:
It's apparently not. Must have been leftover from a previous change.